core/src/main/java/com/statful/client/core/StatfulClientImpl.java

```diff
@@ -93,6 +93,17 @@ public final SenderFacade gauge(final String metricName, final Integer value) {
     }
 
     @Override
+    public final SenderFacade metric(final String metricName, final Integer value) {
```

Comment:
I'm trying to understand the purpose of this method as it isn't on spec. Can you elaborate a bit?